.vscode/debugger_config.yml

```diff
@@ -1 +1,12 @@
 disableStartupPopups: true
+customCommands:
+  - key: 'U'
+    context: 'files'
+    command: 'git add -- CHANGELOG.md && git commit -m "chore: update the changelog"'
+    description: 'Update the Changelog'
+  - key: '<c-p>'
+    context: 'files'
+    command: 'echo "Good work"'
+  - key: '<c-P>'
+    context: 'files'
+    command: 'echo "Good shift work"'
```

Comment:
I suppose you committed this by accident?

Reply:
Yes my bad